frontend/src/modules/SimulationTimeSeries/view.tsx

```diff
@@ -21,6 +21,12 @@ interface MyPlotData extends Partial<PlotData> {
 }
 
 export const view = ({ moduleContext, workbenchSession, workbenchServices }: ModuleFCProps<State>) => {
+    // Leave this in until we get a feeling for React18/Plotly
+    const renderCount = React.useRef(0);
+    React.useEffect(function incrementRenderCount() {
+        renderCount.current = renderCount.current + 1;
+    });
+
```

Comment:
See above. Leave this for now, but consider having a general debug/dev info.

Reply:
👍